### DISCUSSION TITLE
🆕 Enhanced CodeQL Pull Request Alerts Report

### DISCUSSION THREAD

**Discussion by ghostinhershell** [community/community#146856] [Code Security]
[The metrics overview for CodeQL pull request alerts](https://docs.github.com/en/enterprise-cloud@latest/code-security/security-overview/viewing-metrics-for-pull-request-alerts) now includes enhanced tracking and reporting mechanisms, resulting in greater accuracy and more CodeQL pull request alerts and Copilot Autofixes displayed on the dashboard.

These changes retroactively affect the dashboard numbers, allowing you to effectively monitor your organization’s security posture. With these insights, you can proactively identify and address security risks before they reach your default branch.

- **Learn more about this release in the Changelog post here:** [Enhanced CodeQL pull request alerts report](https://github.blog/changelog/2024-12-09-enhanced-codeql-pull-request-alerts-report/).
- **Check out our Docs!** [Viewing metrics for pull request alerts](https://docs.github.com/en/enterprise-cloud@latest/code-security/security-overview/viewing-metrics-for-pull-request-alerts)

**Comment by jsoref** [comment]
Why is this feature limited to Enterprise Cloud?
